backend/tournesol/tests/test_api_statistics.py

```diff
@@ -0,0 +1,133 @@
+
+from datetime import timedelta
+
+from django.test import TestCase
+from django.urls import reverse
+from django.utils import timezone
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from core.models import User
+from tournesol.tests.factories.comparison import ComparisonFactory
+from tournesol.tests.factories.video import VideoFactory
+
+from ..models import Comparison, Video
+
+
+class StatisticsAPI(TestCase):
+    """
+    TestCase of the statistics API.
+    """
+
+    _list_of_users = []
+    _list_of_videos = []
+    _list_of_comparisons = []
+
+    def setUp(self):
+        user_1 = User.objects.create(username="username", email="user@test")
+        user_2 = User.objects.create(
+                username="username2",
+                email="user2@test",
+                date_joined=timezone.now() - timedelta(days=90)
+        )
+        self._list_of_users = [user_1, user_2]
+
+        video_1 = VideoFactory(
+            uploader="uploader1",
+            rating_n_ratings=2,
+        )
+        video_2 = VideoFactory(
+            uploader="uploader2",
+            rating_n_ratings=3,
+        )
+        video_3 = VideoFactory(
+            uploader="uploader2",
+            rating_n_ratings=4,
+        )
+        
+        Video.objects.filter(pk=video_1.pk).update(add_time=timezone.now() - timedelta(days=5))
+        Video.objects.filter(pk=video_2.pk).update(add_time=timezone.now() - timedelta(days=29))
+        Video.objects.filter(pk=video_3.pk).update(add_time=timezone.now() - timedelta(days=60))
+
+        self._list_of_videos = [video_1, video_2, video_3]
+
+        comparison_1 = ComparisonFactory(
+                user=user_1, video_1=video_1, video_2=video_2,
+                duration_ms=102
+        )
+        comparison_2 = ComparisonFactory(
+                user=user_2, video_1=video_1, video_2=video_3,
+                duration_ms=104
+        )
+        comparison_3 = ComparisonFactory(
+                user=user_2, video_1=video_2, video_2=video_3,
+                duration_ms=302
+        )
+
+        Comparison.objects.filter(pk=comparison_1.pk).update(datetime_lastedit=timezone.now() - timedelta(days=5))
+        Comparison.objects.filter(pk=comparison_2.pk).update(datetime_lastedit=timezone.now() - timedelta(days=29))
+        Comparison.objects.filter(pk=comparison_3.pk).update(datetime_lastedit=timezone.now() - timedelta(days=60))
+            
+        self._list_of_comparisons = [comparison_1, comparison_2, comparison_3]
+       
+
+    def test_video_count_is_right(self):
+        """
+        Test if the number of video is right
+        """
+
```

Comment:
I'd like to propose to reword this a bit. The formulation `is_right` doesn't seem to be needed here, as there no test that will ensure something `is_wrong`.

How about something like this?

```suggestion
    def test_video_stats(self):
    """"
    An anonymous user can get statistics about videos.
    """"
```

Reply:
Ok I made the change.